src/MLJInterface.jl

```diff
@@ -83,7 +83,7 @@ MLJModelInterface.@mlj_model mutable struct LGBMRegressor <: MLJModelInterface.D
     metric::Vector{String} = ["l2"]::(all(in.(_, (LGBM_METRICS, ))))
     metric_freq::Int = 1::(_ > 0)
     is_training_metric::Bool = false
-    ndcg_at::Vector{Int} = Vector{Int}([1, 2, 3, 4, 5])::(all(_ .> 0))
```

Comment:
The changes here and in the following structs are due to the alias being used instead of the actual parameter name [reference]((https://lightgbm.readthedocs.io/en/v3.3.5/Parameters.html#eval_at) . The aliases are not supported in LightGBM.jl 
This parameter is only applicable to `ndcg` and `map` metrics which are used in learning to rank applications so I was hesitating whether to remove it completely from Regressors and Classifiers since I created a separate Ranking estimator. It was there for many years so removing would be breaking if someone used these estimators (and technically  eval_at still works with these estimators when the correct metric is used) so I decided to leave it.